rocketpy/plots/rocket_plots.py

```diff
@@ -86,27 +86,31 @@ def stability_margin(self):
 
         return None
 
-    def power_on_drag(self):
-        """Plots power on drag of the rocket as a function of time.
-
-        Returns
-        -------
-        None
-        """
-
-        self.rocket.power_on_drag()
-
-        return None
-
-    def power_off_drag(self):
```

Comment:
With the new behavior, there will be a problem when somebody tries to do these:

```python
my_rocket.plots.power_on_drag()
my_rocket.plots.power_off_drag()
```
Therefore, the current implementation is leading to a small breaking change, although the _RocketPlots is a private class.

I would recommend to keep the old methods but include a warning like this one: https://github.com/RocketPy-Team/RocketPy/blob/435bd62ff2a218825760f73b8bf9fb11c06bbbed/rocketpy/environment/environment.py#L3200

Reply:
Ok. I agree, I think it will be better for advanced users who use the methods.